### ISSUE TITLE
Info 'i' badge doesn't work in Windows high contrast mode - use currentColor?

### ISSUE THREAD

**Issue by petem-ons** [ONSvisual/local-statistics#18]

**Comment by Shiny-Penny** [comment]
<img width="442" height="62" alt="Image" src="https://github.com/user-attachments/assets/e8d39dce-45c9-43f8-a718-8921ce179457" />
